npm/test/upload-form-utils/mockUploadFormDom.ts

```diff
@@ -0,0 +1,249 @@
+import { IReader, IWebkitEntry, UploadForm } from "../../src/upload/upload-form"
```

Comment:
Should rename this file using hypens instead of camel case to keep consistent with other file names: `mock-upload-form-dom.ts`